simulate.py

```diff
@@ -59,7 +59,8 @@ def get_contigs(self):
     parser.add_argument("--max_frag", help="Maximum length of fragment", type=int, default=1000000000)
     parser.add_argument("--min_pass", help="Minimum passes", type=int, default=1)
     parser.add_argument("--max_pass", help="Maximum passes", type=int, default=1000000000)
-    parser.add_argument("--output_polymerase_read", help="shall we simulate polymerase read (including adapter)?", action='store_false')
+    parser.add_argument("--output_polymerase_read", help="shall we simulate polymerase read (including adapter)?", action='store_true')
+    parser.set_defaults(output_polymerase_read=False)
```

Comment:
Thanks! I didn't realize that until I pushed the commit. Shall I PR again? @marghoob